src/Microsoft.Identity.Web/Constants/Constants.cs

```diff
@@ -4,69 +4,102 @@
 namespace Microsoft.Identity.Web
 {
     /// <summary>
-    /// General constants.
+    /// General constants for Microsoft Identity Web.
     /// </summary>
-    internal static class Constants
+    public static class Constants
     {
+        /// <summary>
+        /// LoginHint.
+        /// Represents the preferred_username claim in the ID token.
+        /// </summary>
+        public const string LoginHint = "loginHint";
+
+        /// <summary>
+        /// DomainHint.
+        /// Determined by the tenant Id.
+        /// </summary>
+        public const string DomainHint = "domainHint";
+
+        /// <summary>
+        /// Claims.
+        /// Determined from the signed-in user.
+        /// </summary>
+        public const string Claims = "claims";
+
+        /// <summary>
+        /// Bearer.
+        /// Predominant type of access token used with OAuth 2.0.
+        /// </summary>
+        public const string Bearer = "Bearer";
+
+        /// <summary>
+        /// AzureAd.
+        /// Configuration section name for AzureAd.
+        /// </summary>
+        public const string AzureAd = "AzureAd";
+
+        /// <summary>
+        /// AzureAdB2C.
+        /// Configuration section name for AzureAdB2C.
+        /// </summary>
+        public const string AzureAdB2C = "AzureAdB2C";
+
+        /// <summary>
+        /// Scope.
```

Comment:
I don't know what else to say here....to not confuse it w/Scopes or string[] scopes.